src/components/Map/VectorTileLayer.tsx

```diff
@@ -27,12 +27,17 @@ export default function VectorTileLayer({ countries, disputedAreas, ipcData, nut
       { countries, disputedAreas, ipcData, selectedMapType, nutritionData },
       mapContainer
     );
-    const popover = new mapboxgl.Popup({ closeButton: false, closeOnClick: false });
+    const popover = new mapboxgl.Popup({
+      closeButton: false,
+      closeOnClick: false,
+      anchor: 'bottom',
```

Comment:
maybe also change anchor to left/right to stay consistent with other map types

Reply:
I ve set the alignment to 'left.' Automatically switching between left and right would require quite a bit of custom code since this isn’t supported by Mapbox by default. https://docs.mapbox.com/mapbox-gl-js/api/markers/#popup

Is this okay for you? If not, I m happy to implement the "switiching", just wanted to point this out beforehand.